unlock-js/src/utils.js

```diff
@@ -17,6 +18,11 @@ export default {
       .formatUnits(ethers.utils.bigNumberify(num), units)
       .replace(/\.0$/, '')
   },
+  fromDecimal: (num, decimals) => {
+    return ethers.utils
+      .formatUnits(ethers.utils.bigNumberify(num), decimals)
+      .replace(/\.0$/, '')
```

Comment:
Is there not a way to have `formatUnits` return an integer?

Reply:
I wanted to maintain consistency with the existing `toWei` and `fromWei` function.

Reply:
from https://docs.ethers.io/ethers.js/html/api-utils.html

"Format an amount of wei into a decimal string representing the amount of units. The output will always include at least one whole number and at least one decimal place, otherwise leading and trailing 0’s will be trimmed." `formatUnits` annoyingly always includes a decimal place

Reply:
which do the same thing but for ether decimals (wei, gwei, szabo, .... etc)